### PR TITLE
Add command to submit validation data & aggregate validation status

### PR DESCRIPTION
**Pull request by kathy-t**
[SEAB-4695](https://ucsc-cgl.atlassian.net/browse/SEAB-4695)

View https://github.com/dockstore/dockstore/pull/5447 for the webservice changes

This PR adds a command to the metrics aggregator to submit validation data so we can add DNAstack's validation information to Dockstore. The command takes in a CSV field containing the TRS ID, version name, isValid boolean, and the date the validation was executed. I created a script that transforms the file DNAstack gave us into a CSV file. I ran the `submit-validation-data` command with the CSV file and it took about 3-4 minutes.

This PR also aggregates the validation execution data found in S3.

**Review Instructions**
- SSM into the deployer
- git clone the dockstore-support repo and do cd metricsaggregator && mvn clean install -DskipTests
- Create a configuration file as defined in the metricsaggregator README. Do not provide an S3 endpoint override. Example:
   ```
  [dockstore]
  server-url: https://qa.dockstore.org/api
  token: <Dockstore token of a curator or admin>
  
  [s3]
  bucketName: qa-dockstore-metrics-data
   ```
- Download the `workflow_names.txt` file in the ticket and put this on the Deployer. This contains workflows that validated correctly with miniwdl.
- Locate the JAR and run the java program to submit successful validation data for miniwdl. Example instructions are in the README. It should not fail.
- Run the java program to aggregate the metrics. It should not fail. Verify that there are validation metrics for some of the workflows aggregated.

[SEAB-4695]: https://ucsc-cgl.atlassian.net/browse/SEAB-4695?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by coverbeck — COMMENTED** [review]
Was here, going to wait for revisions (if any?) after our huddle.

File: metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/AggregationHelper.java [hunk 1]
```diff
@@ -140,4 +156,25 @@ public static Optional<MemoryMetric> getAggregatedMemory(List<Execution> executi
         }
         return Optional.empty();
     }
+
+    /**
+     * Aggregate Validation metrics from the list of validation executions by performing a logical AND.
```
Comment (denis-yuen) [inline metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/AggregationHelper.java:161]: Ditto, I think this is the section which will need a few more additional statistics IIRC

File: metricsaggregator/src/test/resources/miniwdl-successful-validation-workflow-names.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+trsId,versionName,isValid,dateExecuted
+#workflow/github.com/garyluu/testWorkflow
```
Comment (coverbeck) [inline metricsaggregator/src/test/resources/miniwdl-successful-validation-workflow-names.txt:2]: Is this missing 2 column values on purpose (may be in a test, haven't looked).

**Review by coverbeck — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

File: metricsaggregator/src/test/resources/miniwdl-successful-validation-workflow-names.txt (hunk 2 above)
Comment (kathy-t) [reply metricsaggregator/src/test/resources/miniwdl-successful-validation-workflow-names.txt:2]: Yeah, it's to verify that the client doesn't send data to Dockstore for lines missing fields

**Review by svonworl — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Pull request merged**